packages/island/src/theme-default/layout/Layout/index.tsx

```diff
@@ -53,7 +64,7 @@ export const Layout: React.FC<LayoutProps> = (props) => {
   const getContentLayout = () => {
     switch (pageType) {
       case 'home':
-        return <HomeLayout />;
+        return <HomeLayout {...homeProps} />;
```

Comment:
Do you use the props in HomeLayout component?

Reply:
Didn't the HomeLayout component use these props?:confused:

https://github.com/sanyuan0704/island.js/blob/master/packages/island/src/theme-default/layout/HomeLayout/index.tsx#L13